CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to this project will be documented in this file.
 ### New Features
 
 - Add `getUserGasPrice` method to `Web3jAbstractService`. (#104)
+- Use `poco-chain` with `poco v5.5.0` and `voucher v1.0.0` in tests (#106)
```

Comment:
Same remark as in the other PR.
Business code (`main` source set) is not modified here.
I think it would be better to add the **Quality** section between New Features and **Dependency Upgrades**.